tests/integration/test_rex.py

```diff
@@ -926,8 +926,8 @@ def test_timestampdiff(c):
             "hours": [46, 6648, 591744],
             "days": [1, 277, 24656],
             "weeks": [0, 39, 3522],
-            "months": [0, 9, 810],
-            "quarters": [0, 3, 270],
+            "months": [0, 9, 821],
```

Comment:
Since this behavior is different from the results we get from mysql and a few implementations that support timestampdiff, I'm not sure whether we want to keep the existing logic or modify it. I didn't find an easy way to get a timestampdiff with datafusion/postgresql